x/distribution/types/distribution.pb.go

```diff
@@ -8,7 +8,7 @@ import (
 	_ "github.com/cosmos/cosmos-proto"
 	github_com_cosmos_cosmos_sdk_types "github.com/cosmos/cosmos-sdk/types"
 	types "github.com/cosmos/cosmos-sdk/types"
-	_ "github.com/gogo/protobuf/gogoproto"
+	_ "github.com/cosmos/gogoproto/gogoproto"
```

Comment:
Question: Is this expected to change? Seems weird to me. I used ` make proto-gen`.

Reply:
Which version of proto-builder are you using?

Reply:
It usually means you are using the wrong version, latest uses the latest `gogo.proto`

https://buf.build/cosmos/gogo-proto/file/5e5b9fdd01804356895f8f79a6f1ddc1:gogoproto/gogo.proto#L36 
vs
https://buf.build/cosmos/gogo-proto/file/bee5511075b7499da6178d9e4aaa628b:gogoproto/gogo.proto#L36

Reply:
> Which version of proto-builder are you using?

Note that I only used make `proto-gen` which pulls docker images as you know. 

If I prune all images locally and start from scratch, it starts with: 
```
Generating Protobuf files
Unable to find image 'tendermintdev/sdk-proto-gen:v0.7' locally
v0.7: Pulling from tendermintdev/sdk-proto-gen
```

Reply:
Note that this is the same version as in the latest upstream SDK 0.46.15: https://github.com/cosmos/cosmos-sdk/blob/d6d929843bbd331b885467475bcb3050788e30ca/Makefile#L383-L384

Reply:
The only diff here is that our fork's go.mod differs here:
https://github.com/celestiaorg/cosmos-sdk/blob/64adc6e40ae679003d2f779387e06c4105c99121/go.mod#L20
this dep doesn’t exist in upstream:
https://github.com/cosmos/cosmos-sdk/blob/d6d929843bbd331b885467475bcb3050788e30ca/go.mod#L15-L25

same for 0.46.14 btw:
https://github.com/cosmos/cosmos-sdk/blob/v0.46.14/go.mod

Reply:
@julienrbrt is there anything I should be doing differently?

Reply:
@evan-forbes @rootulp @cmwaters any ideas about the above diff?

Reply:
> @julienrbrt is there anything I should be doing differently?

No, if it uses the right proto-builder image (with the right gogo.proto), then it should not be adding a cosmos/gogoproto dependency, so I am not sure here.

Reply:
FYI, this fixed it: https://github.com/celestiaorg/cosmos-sdk/pull/343